react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js

```diff
@@ -1,478 +1,209 @@
-            noMatches = 'addPeople.noResults';
-            placeholder = 'addPeople.disabled';
-        }
-
-        if (_footerTextEnabled) {
-            footerText = {
-                content: <div className = 'footer-text-wrap'>
-                    <div>
-                        <span className = 'footer-telephone-icon'>
-                            <Icon src = { IconPhone } />
-                        </span>
-                    </div>
-                    { translateToHTML(t, 'addPeople.footerText') }
-                </div>
-            };
-        }
-
-        return (
-            <Dialog
-                okDisabled = { this._isAddDisabled() }
-                okKey = 'addPeople.add'
-                onSubmit = { this._onSubmit }
-                titleKey = 'addPeople.title'
-                width = 'medium'>
-                <div className = 'add-people-form-wrap'>
-                    { this._renderErrorMessage() }
-                    <MultiSelectAutocomplete
-                        footer = { footerText }
-                        isDisabled = { isMultiSelectDisabled }
-                        loadingMessage = { t(loadingMessage) }
-                        noMatchesFound = { t(noMatches) }
-                        onItemSelected = { this._onItemSelected }
-                        onSelectionChange = { this._onSelectionChange }
-                        placeholder = { t(placeholder) }
-                        ref = { this._setMultiSelectElement }
-                        resourceClient = { this._resourceClient }
-                        shouldFitContainer = { true }
-                        shouldFocus = { true } />
-                </div>
-            </Dialog>
-        );
-    }
-
-    _invite: Array<Object> => Promise<*>
-
-    _isAddDisabled: () => boolean;
-
-    _onItemSelected: (Object) => Object;
+    t: Function,
 
     /**
-     * Callback invoked when a selection has been made but before it has been
-     * set as selected.
-     *
-     * @param {Object} item - The item that has just been selected.
-     * @private
-     * @returns {Object} The item to display as selected in the input.
+     * Method to update the dial in numbers.
      */
-    _onItemSelected(item) {
-        if (item.item.type === 'phone') {
-            item.content = item.item.number;
-        }
-
-        return item;
-    }
+    updateNumbers: Function
+};
 
-    _onSelectionChange: (Map<*, *>) => void;
+/**
+ * Invite More component.
+ *
+ * @returns {React$Element<any>}
+ */
+function AddPeopleDialog({
+    conferenceName,
+    dialIn,
+    hideInviteContacts,
+    inviteUrl,
+    liveStreamViewURL,
+    localParticipantName,
+    locationUrl,
+    t,
+    updateNumbers }: Props) {
+    const [ phoneNumber, setPhoneNumber ] = useState(undefined);
+
+    /**
+     * Updates the dial-in numbers.
+     */
+    useEffect(() => {
+        if (!dialIn.numbers) {
+            updateNumbers();
+        }
+    }, []);
```

Comment:
This effect should filter by the dialIn numbers, right?

Reply:
Not in this specific case, this used to be called when the `InviteToolbarButton` mounted, so I moved it here, when the actual dialog mounts.